crates/biome_html_analyze/src/lint/nursery/use_vue_consistent_v_bind_style.rs

```diff
@@ -0,0 +1,183 @@
+                None
+            }
+            _ => None,
+        }
+    }
+
+    fn diagnostic(ctx: &RuleContext<Self>, state: &Self::State) -> Option<RuleDiagnostic> {
+        let prefer = ctx.options().style();
+        let message = match (state, prefer) {
+            (AnyVueDirective::VueDirective(_), VueDirectiveStyle::Shorthand) => {
+                markup! { "Use shorthand ':' syntax instead of v-bind." }
+            }
+            (AnyVueDirective::VueVBindShorthandDirective(_), VueDirectiveStyle::Longhand) => {
+                markup! { "Use longhand 'v-bind' syntax instead of ':'." }
+            }
+            _ => {
+                // should be unreachable, but just in case
+                debug_assert!(
+                    false,
+                    "Diagnostic should only be created for invalid states."
+                );
+                return None;
+            }
+        };
+        let note = match (state, prefer) {
+            (AnyVueDirective::VueDirective(_), VueDirectiveStyle::Shorthand) => {
+                markup! { "This project prefers to use shorthand syntax for v-bind." }
+            }
+            (AnyVueDirective::VueVBindShorthandDirective(_), VueDirectiveStyle::Longhand) => {
+                markup! { "This project prefers to use longhand syntax for v-bind." }
+            }
+            _ => {
+                // should be unreachable, but just in case
+                debug_assert!(
+                    false,
+                    "Diagnostic should only be created for invalid states."
+                );
+                return None;
+            }
+        };
+        Some(RuleDiagnostic::new(rule_category!(), state.range(), message).note(note))
+    }
+
+    fn action(ctx: &RuleContext<Self>, state: &Self::State) -> Option<crate::HtmlRuleAction> {
+        let prefer = ctx.options().style();
+        let mut mutation = biome_rowan::BatchMutationExt::begin(ctx.root());
+        match (state, prefer) {
+            // Convert longhand v-bind:prop to :prop
+            (AnyVueDirective::VueDirective(dir), VueDirectiveStyle::Shorthand) => {
+                if let Some(arg) = dir.arg() {
+                    let mut builder = make::vue_v_bind_shorthand_directive(arg, dir.modifiers());
+                    if let Some(init) = dir.initializer() {
+                        builder = builder.with_initializer(init);
+                    }
+                    let new_node = builder.build();
+                    mutation.replace_node(
+                        AnyVueDirective::VueDirective(dir.clone()),
+                        AnyVueDirective::VueVBindShorthandDirective(new_node),
+                    );
+                }
+                Some(biome_analyze::RuleAction::new(
+                    ctx.metadata().action_category(ctx.category(), ctx.group()),
+                    ctx.metadata().applicability(),
+                    markup! { "Remove v-bind to use the shorthand ':' syntax." }.to_owned(),
```

Comment:
Can we be consistent in the phrasing of both code suggestions? E.g. "Use the <STYLE> syntax instead."